test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -161,7 +161,8 @@ Describe "Get-ChildItem" -Tags "CI" {
             (Get-ChildItem -Path $searchRoot -Directory -Recurse).Count | Should -Be 1
         }
 
-        It "Should give .sys file if the fullpath is specified with hidden and force parameter" -Skip:(!$IsWindows) {
+        # VSTS machines don't have a page file
```

Comment:
at all or on system drive?

Reply:
They don't have a page file on the system drive

Reply:
We could find the page file location:
```powershell
			#Open the registry on multiple remote computers
			$reg = [Microsoft.Win32.RegistryKey]::OpenRemoteBaseKey('LocalMachine',$server )
			$RegKeyPath= "SYSTEM\CurrentControlSet\Control\Session Manager\Memory Management"
			$pageFileKey=$reg.OpenSubKey($RegKeyPath)
			$pageFileLocation=$pageFileKey.GetValue("ExistingPageFiles")
```